product_docs/docs/migration_portal/4/known_issues_notes.mdx

```diff
@@ -231,6 +231,20 @@ While using the Oracle default case, you may experience a lower compatibility ra
    Character: 58
    ```
 
+### Incomplete Object Definition
+
+While assessing source schemas with Migration Portal, Object Definitions can be incomplete or the DDL assessment is terminated when the source DDL contains reserved words as object names.
+
+If you encounter an issue where some part of the Object Definition is missing or the definition gets terminated before reaching the actual end of the DDL, you are possibly using keywords reserved by EDB Postgres Advanced Server. For example, when you use `CONSTRAINT` as a column name and then use it in a `TABLE`, `VIEW` or `MVIEW` definition, the definition is terminated before reaching the end of the DDL.
+
+To work around this issue, you have these options: 
+
+- Rename the column name in the Oracle database, then reextract the DDL and upload it to Migration Portal.
+
+or
+
+- Rename the column name in the extracted DDL file, and then upload it to Migration Portal.
+
```

Comment:
I am wondering if another workaround is to simply update the target DDL with a copy of the source DDL for the object and manually repair that particular object directly in the Migration Portal.  I left a similar comment on the Jira ticket ([MIG-5001](https://enterprisedb.atlassian.net/browse/MIG-5001)).

Reply:
From what I can see in the Jira ticket, @umangjtank replied that this suggestion, unfortunately, won't fix the issue. 
I have corrected the other instances and think we are good to go. Can I get an approval, please?